modules/cli/pages/index.adoc

```diff
@@ -178,6 +178,56 @@ $ skupper link delete <link-name>
 ----
 where `<link-name>` is the name of the link specified during creation.
 
+// Type: procedure
+[id="link-cost"] 
+== Specifying link cost
+
+When linking sites, you can assign a cost to each link to influence the traffic flow. By default, link cost is set to `1` for a new link.
+
+If you have more than two sites and services distributed across different clusters, you might want a calling service to favor a particular target or link.
+In this case, you can specify a cost of greater than `1` for links that should be avoided.
+
+.Prerequisite
+
+* You have set your Kubernetes context to a site that you want to link _from_.
+* A token for the site that you want to link _to_.
+
+.Procedure
+
+. Create a link to the {service-network}:
++
+--
+----
+$ skupper link create <filename> --cost <integer-cost>
+----
+
+where `<integer-cost>` is an integer greater than 1 and traffic favors lower cost links.
```

Comment:
Propose adding a comment along the lines of:

"It should be noted that service targets local to a skupper-router have an implicit cost of `0`. "

Meaning there is a bias to use a local service over one across a link.

Reply:
added note below